readme.md

```diff
@@ -1243,6 +1243,32 @@ export default () => <div>
 </div>
 ```
 
+#### Configuring Babel env preset
+
+Next.js uses the env preset under the hood, and you can passover all the parameters to the next.js without duplicating the env preset in your `babel.rc`.
+The `next/babel` preset has a second parameter, `preset-env`, which allows custimization of the env preset. 
+
+```json
+{
+  "presets": [[
+    "next/babel", {
+      "preset-env": {
+        "targets": {
+          "safari": 9,
```

Comment:
Might be good to mention that `babel-present-env` uses `browserslist` and thus can use any of the standard config file locations such as `package.json` with `browserslist` key.